{{cookiecutter.project_slug}}/.env-example

```diff
@@ -0,0 +1,31 @@
+// DB
```

Comment:
why did you move this inside the project?

Reply:
It was your idea as far as I remember, to fill the envs directly here and read it frm the vscode configs